src/test/java/com/sproutsocial/nsq/RoundRobbinIT.java

```diff
@@ -0,0 +1,130 @@
+package com.sproutsocial.nsq;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+//run one test: mvn "-Dit.test=PublishIT#testSnappy" verify
+
+public class RoundRobinIT extends SubscribeBase {
```

Comment:
the class is named `RoundRobinIT` but the file is named `RoundRobbinIT` with two b's